girder/conf/girder.dist.cfg

```diff
@@ -20,6 +20,10 @@ replica_set = None
 mode = "development"
 api_root = "api/v1"
 
+# Disable this if operating in a WSGI environment where Girder is unable
+# to restart itself
+restartable = True
```

Comment:
Maybe we should make this setting more generic, like `cherrypy_server = True`. What do you think?